forms/contributor-signup.md

```diff
@@ -0,0 +1,40 @@
+Google form link: to be created
+
+# Mentored Sprint at PyCon US: Contributor Sign Up
+
+Please fill in this form if you would like to be mentored during the sprint.
+
+For more information, please check: https://us.pycon.org/2019/hatchery/mentoredsprints/
+
+1. Email address (required)
+
+2. Your name (required)
+
+3. Are you a member of underrepresented group?
+
+4. Preferred open source projects to contribute to (should this be required?)
```

Comment:
I do not think this should be required, but could perhaps rephrase to something like:

4. Have you got some projects in mind you would like to contribute to?